engine-rest/engine-rest-openapi/src/main/templates/paths/process-instance/suspended-async/post.ftl

```diff
@@ -8,7 +8,7 @@ a process instance query, and/or a historical process instance query.",
 
   <@lib.requestBody
       mediaType = "application/json"
-      dto = "ProcessInstanceSuspensionStateQueriesDto" />
+      dto = "ProcessInstanceSuspensionStateDto" />
```

Comment:
Here we should either mention which parameters of the DTO do not work for async, or we should pull the DTO apart into two, one for async and one for sync (in both implementation and OpenAPI description).